packages/ml_api/tox.ini

```diff
@@ -22,13 +22,6 @@ deps =
 passenv =
       {[testenv]passenv}
 
-setenv =
-  PYTHONPATH=.
-  DB_USER={env:DB_USER:test_davit}
-  DB_PASSWORD={env:DB_PASSWORD:19dv009}
-  DB_HOST={env:DB_HOST:localhost}
-  DB_NAME={env:DB_NAME:ml_api_test}
-
```

Comment:
Oops... yes, it will break the code...

I have 2 questions.
1. Do I get it right that tox is independent of docker?  I think it's not ok...
2. Why do you want to remove default values? And how is this related to my PR? :)

Reply:
Answering your questions:
1. I am not sure what you mean by independence but if I understand you correctly, then my answer is that tox and docker are NOT independent. tox runs some tests that make requests to our endpoints and for running our server we use docker, so if docker is not running tests will fail.
2. We don't want to have hardcoded secret information in the code, that is why I don't want to have any default values. What refers to your PR, I thought you want to refactor the code parts related to environment variables :) Since you started it, I want you to fix all parts.